pkgs/tools/misc/graylog/3.3.nix

```
@@ -0,0 +1,9 @@
+{ callPackage, lib, ...}:
```

```suggestion
{ callPackage, lib }:
```

there are no extra variables we could ignore here